rho_playbook.yml

```diff
@@ -16,7 +16,6 @@
     - dmi
     - etc_release
     - file_contents
-    - fuse_on_karaf
```

Comment:
Didn't mean for you to remove the role. Just rename it and have it located with `jboss_eap` in the playbook.
Example
```
- jboss_eap
- jboss_eap_scan
- brms
- fuse
- jboss_fuse_on_karaf
```